dev/tools/github-check-prs.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
```

Comment:
What is this file?

Reply:
Hmmm, it's in master, not sure why it shows up in the PR. Will fix.